src/main/java/io/confluent/connect/jdbc/dialect/GenericDatabaseDialect.java

```diff
@@ -1096,6 +1096,7 @@ protected String addFieldToSchema(
         glog.debug("DECIMAL with precision: '{}' and scale: '{}'", precision, scale);
         scale = decimalScale(columnDefn);
         SchemaBuilder fieldBuilder = Decimal.builder(scale);
+        fieldBuilder.parameter("connect.decimal.precision", Integer.toString(precision));
```

Comment:
nit: move `connect.decimal.precision` to a private static final String.

Reply:
Okay, updated (using the variable name used in ksql: https://github.com/confluentinc/ksql/blob/d15026fe7c8993673152d740d46b16253896d8e8/ksql-common/src/main/java/io/confluent/ksql/util/DecimalUtil.java#L33)

Thanks!